schema/patches/3.sql

```diff
@@ -25,14 +29,16 @@ AS $$
         officerships INTEGER[];
         num_shows INTEGER;
     BEGIN
-        -- Find officerships
+        -- Find the officerships this member currently has
+	--  Checks both analytics history and officers with this memberid
```

Comment:
Not quite - `create_analytics_record` is simply the name of the function, so all `create_analytics_record.memberid` does is reference the `memberid` parameter (prefixed with the function name otherwise `memberid` is ambiguous).